src/chm/documents/customactions/wixnetfxextension.html.md

```diff
@@ -7,122 +7,6 @@ after: using_standard_customactions
 
 The [WixNetfxExtension](~/xsd/netfx/index.html) includes a set of custom actions to compile native images using Ngen.exe. For an example, see [How To: NGen managed assemblies during installation](~/howtos/files_and_registry/ngen_managed_assemblies.html).
 
-## PackageGroups
```

Comment:
I think reference material like this should stay with its "owning" topic. How-tos are less discoverable.